src/Oro/Bundle/EmbeddedFormBundle/Resources/public/js/embed.form.js

```diff
@@ -79,7 +79,8 @@ var ORO = (function(ORO) {
                     realTag = document.createElement('script');
 
                     if (scripts[k].hasAttribute('src')) {
-                        realTag.setAttribute('src', scripts[k].getAttribute('src'));
+                        realTag.async = scripts[k].async;
+                        realTag.src = scripts[k].src;
                     } else {
                         realTag.innerHTML = scripts[k].innerHTML;
                     }
```

Comment:
Let's improve a bit the solution to cover the case if some other attribute is required in `realTag` (e.g. `type` or `id`):
```js
    for (var i = 0; i < scripts[k].attributes.length; i++) {
        realTag.setAttribute(scripts[k].attributes.name, scripts[k].attributes.value);
    }

    if (!scripts[k].hasAttribute('src')) {
        realTag.innerHTML = scripts[k].innerHTML;
    }
```
And let's copy attribute even if it's inline script (e.g. in case the inline script has `type="text/template"`)

Thanks.

Reply:
Also attribute copying should be placed outside `if (scripts[k].hasAttribute('src')) {` 
statement as inline scripts also can have attributes

Reply:
Good idea :+1: , it looks better, I fixed it in PR
Thanks